src/components/common/FilterDropdownMenu.jsx

```diff
@@ -0,0 +1,28 @@
+import Form from 'react-bootstrap/Form';
+import React, { useState } from 'react';
+
+const CustomMenu = React.forwardRef(
+  ({ children, style, className, 'aria-labelledby': labeledBy }, ref) => {
+    const [value, setValue] = useState('');
+
+    return (
+      <div ref={ref} style={style} className={className} aria-labelledby={labeledBy}>
+        <Form.Control
+          autoFocus
+          className="mx-3 my-2 w-auto"
+          placeholder="Type to filter..."
+          onChange={(e) => setValue(e.target.value)}
```

Comment:
nit, we can [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) this for better ux

```suggestion
          onChange={(e) => setValue(e.target.value.trim())}
```